transformer_engine/jax/fp8.py

```diff
@@ -385,3 +386,27 @@ def update_fp8_metas(state: Collection) -> Collection:
         The collection with updated FP8 metas.
     """
     return FP8Helper.update_fp8_metas(state)
+
+
+def get_delayed_scaling():
+    r"""
+    Obtain an instance of  DelayedScaling which is set via fp8_autocast.
+
+    .. note::
+        We only store :attr:`margin`, :attr:`fp8_format`, :attr:`interval`,
+        :attr:`amax_history_len` and :attr:`amax_compute_algo` via fp8_autocast.
+        Other parameters in recipe.DelayedScaling would be returned as the default
+        values.
```

Comment:
Why? What is the reason we are not storing the actual Python object for the used recipe?

Reply:
It is kind of a legacy issue, will try to store actual DelayedScaling in the near feature.

Reply:
This issue also can be solved in next PR